assets/stylesheets/_z-index.scss

```diff
@@ -70,6 +70,9 @@ $z-layers: (
 	".edit-post-sidebar": 100000,
 	".edit-widgets-sidebar": 100000,
 	".edit-post-layout .edit-post-post-publish-panel": 100001,
+	// For larger views, the wp-admin navbar dropdown should be at top of
+	// the Publish Post sidebar.
+	".edit-post-layout .edit-post-post-publish-panel-break-medium": 99998,
```

Comment:
Should we expand the comment and say something like value should be bellow: #wpadminbar { z-index: 99999 }?